src/test/java/ca/uhn/fhir/jpa/starter/service/HelperServiceTest.java

```diff
@@ -0,0 +1,759 @@
+		field.setAccessible(true);
+		Map<String, DashboardConfigContainer> mapToReturn = new HashMap<>();
+		field.set(helperService, mapToReturn);
+
+		// Call the method and assert the response for a file not found scenario
+		ResponseEntity<?> responseEntity = helperService.getIndicators("NonExistentEnv");
+
+		// Assertions for a file not found scenario
+		assertEquals(HttpStatus.OK, responseEntity.getStatusCode());
+		assertEquals("Error : ScoreCard Config File Not Found", responseEntity.getBody());
+	}
+
+	@Test
+	void getCategories_Success() throws IOException, NoSuchFieldException, IllegalAccessException {
+		// Mock data
+		CategoryItem  mockIndicators = readJsonFile("CATEGORY_DEFINITIONS.json", CategoryItem.class);
+
+		// Create an instance of DashboardConfigContainer
+		DashboardConfigContainer container = new DashboardConfigContainer();
+
+		// Assign values to its fields
+		container.setCategoryItem(mockIndicators);
+
+		// Set the private field in helperService
+		Field field = HelperService.class.getDeclaredField("dashboardEnvToConfigMap");
+		field.setAccessible(true);
+		Map<String, DashboardConfigContainer> mapToReturn = new HashMap<>();
+		mapToReturn.put("V2", container);
+		field.set(helperService, mapToReturn);
+
+		// Call the method and assert the response
+		ResponseEntity<?> responseEntity = helperService.getCategories("V2");
+
+		// Assertions for a successful scenario
+		assertEquals(HttpStatus.OK, responseEntity.getStatusCode());
+		assertEquals(mockIndicators, responseEntity.getBody());
+	}
+
+	@Test
+	void getCategories_FileNotFound() throws IOException, NoSuchFieldException, IllegalAccessException {
+		// Set the private field in helperService to simulate an empty configuration
+		Field field = HelperService.class.getDeclaredField("dashboardEnvToConfigMap");
+		field.setAccessible(true);
+		Map<String, DashboardConfigContainer> mapToReturn = new HashMap<>();
+		field.set(helperService, mapToReturn);
+
+		// Call the method and assert the response for a file not found scenario
+		ResponseEntity<?> responseEntity = helperService.getCategories("NonExistentEnv");
+
+		// Assertions for a file not found scenario
+		assertEquals("Error : Category Config File Not Found", responseEntity.getBody());
+	}
+
+
+
+	@Test
+	void getEnvironmentOptions_Success() throws IOException, NoSuchFieldException, IllegalAccessException {
+		CategoryItem  mockIndicators = readJsonFile("CATEGORY_DEFINITIONS.json", CategoryItem.class);
+
+		// Create an instance of DashboardConfigContainer
+		DashboardConfigContainer container = new DashboardConfigContainer();
+
+		// Assign values to its fields
+		container.setCategoryItem(mockIndicators);
+
+		// Set the private field in helperService
+		Field field = HelperService.class.getDeclaredField("dashboardEnvToConfigMap");
+		field.setAccessible(true);
+		Map<String, DashboardConfigContainer> mapToReturn = new HashMap<>();
+		mapToReturn.put("V2", container);
+		field.set(helperService, mapToReturn);
```

Comment:
Do you think this can be moved to a function that runs before each test with the @Before annotation?